src/common/options.cc

```diff
@@ -445,7 +445,7 @@ std::vector<Option> get_global_options() {
     Option("container_image", Option::TYPE_STR, Option::LEVEL_BASIC)
     .set_description("container image (used by cephadm orchestrator)")
     .set_flag(Option::FLAG_STARTUP)
-    .set_default("docker.io/ceph/ceph:v15"),
+    .set_default("registry.suse.com/ses/7/ceph/ceph"),
```

Comment:
Should this have a default tag?

Reply:
Yeah, I reckon it should be registry.suse.com/ses/7/ceph/ceph:latest, so it's exactly the same as DEFAULT_IMAGE in src/cephadm/cephadm (https://github.com/SUSE/ceph/blob/c31b8bd85339701a1ea5eade04e3dd2e92402cba/src/cephadm/cephadm#L3)